toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -276,16 +321,38 @@ func (im *IsoMaker) prepareWorkDirectory() {
 // copyStaticIsoRootFiles copies architecture-independent files from the
 // Mariner repo directories.
 func (im *IsoMaker) copyStaticIsoRootFiles() {
-	staticIsoRootFilesPath := filepath.Join(im.resourcesDirPath, "assets/isomaker/iso_root_static_files/*")
 
-	logger.Log.Debugf("Copying static ISO root files from '%s'.", staticIsoRootFilesPath)
+	if im.resourcesDirPath == "" && im.grubCfgPath == "" {
+		logger.Log.Panicf("Missing required parameters. Must specify either the resources directory or provide a grub.cfg.")
+	}
+
+	if im.resourcesDirPath != "" {
+		staticIsoRootFilesPath := filepath.Join(im.resourcesDirPath, "assets/isomaker/iso_root_static_files/*")
+
+		logger.Log.Infof("Copying static ISO root files from '%s' to '%s'.", staticIsoRootFilesPath, im.buildDirPath)
```

Comment:
Why do we want this log on info level? If I recall correctly, it contains paths from toolkit's internals. Not really something a regular user cares about. I'd prefer to keep such internal stuff on debug level to minimize noise.

Reply:
Not needed. Changed back to Debugf.